### PR TITLE
ACN_CD_VMwareESXi_DataConnector01

### PR DESCRIPTION
**Pull request by chicduong**
## Proposed Changes

  - VMware ESXi data connector page
  - VMware ESXi Sample data

### CODE REVIEW

File: DataConnectors/VMware ESXi/Connector_Syslog_VMwareESXi.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    "id": "VMwareESXi",
+    "title": "VMWare ESXi", 
+    "publisher": "VMWare", 
+    "descriptionMarkdown": "The [VMWare ESXi](https://www.vmware.com/products/esxi-and-esx.html) connector allows you to easily connect your VMWare ESXi logs with Azure Sentinel This gives you more insight into your organization's ESXi servers and improves your security operation capabilities.", 
+    "additionalRequirementBanner":"These queries and workbooks are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **VMWareESXi** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinelgithubparsersvmwareesxi)",
+    "graphQueries": [ 
+     { 
+         "metricName": "Total data received", 
+         "legend": "VMwareESXi", 
+         "baseQuery": "VMwareESXi"
```
Comment (v-jayakal) [inline DataConnectors/VMware ESXi/Connector_Syslog_VMwareESXi.json:11]: Kusto Function name is VMWareESXi, Please correct- w in VMwareESXi is small, hence resulting in error. Please fix in all queries.

File: DataConnectors/VMware ESXi/Connector_Syslog_VMwareESXi.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+     } 
+    ], 
+          "sampleQueries": [ 
+           { 
+               "description" : "Total Events by Log Type", 
+               "query": "VMwareESXi \n| summarize count() by ProcessName"    
+           },
+        {
+            "description" : "Top 10 ESXi Hosts Generating Events",
+            "query": "VMwareESXi /n | summarize count() by HostName /n | top 10 by count_"
```
Comment (v-jayakal) [inline DataConnectors/VMware ESXi/Connector_Syslog_VMwareESXi.json:21]: \n is correct for character, not  /n. Please fix.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/VMware ESXi/Connector_Syslog_VMwareESXi.json (hunk 1 above)
Comment (chicduong) [reply DataConnectors/VMware ESXi/Connector_Syslog_VMwareESXi.json:11]: corrections have been made

File: DataConnectors/VMware ESXi/Connector_Syslog_VMwareESXi.json (hunk 2 above)
Comment (chicduong) [reply DataConnectors/VMware ESXi/Connector_Syslog_VMwareESXi.json]: corrections have been made

**Review by v-jayakal — APPROVED** [review]

**Review by nazang — APPROVED** [review]

**Pull request merged**